compiler/docgen.nim

```diff
@@ -886,7 +892,12 @@ proc generateDoc*(d: PDoc, n, orig: PNode) =
     for it in n: traceDeps(d, it)
   of nkExportStmt:
     for it in n:
-      if it.kind == nkSym: exportSym(d, it.sym)
+      if it.kind == nkSym:
+        let ast = it.sym.ast
+        if ast != nil and it.info.fileIndex == ast.info.fileIndex:
```

Comment:
No, but close. Check the it.sym.owner field instead.

Reply:
done; had to add some logic to expose `module`